src/main/java/csci432/ItemTracker.java

```diff
@@ -0,0 +1,97 @@
+package csci432;
+import java.util.concurrent.*;
+import java.util.*;
+import java.awt.image.BufferedImage;
+
+/**
+ * Class used for identifying moving objects, called blobs, based off
+ * of images that have been passed through Sigma-Delta and edge
+ * detection algorithms.
+ *
+ **/
+public class ItemTracker<T> {
+    BlockingQueue<BufferedImage> queue;
```

Comment:
The blocking methods aren't included in the Queue interface. Or is it too early to think about it?